commit 17c4da1ecfc18ac07c14a7060b1a3841e385ad5a
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Fri Oct 8 14:11:20 2010 -0700

    Staging: brcm80211: remove BCMATTACHDATA macro
    
    It's not even used for anything, not to mention, it is pretty silly.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/util/bcmsrom.c
+++ b/drivers/staging/brcm80211/util/bcmsrom.c
@@ -253,125 +253,125 @@ srom_read(si_t *sih, uint bustype, void
 	return 0;
 }
 
-static const char BCMATTACHDATA(vstr_manf)[] = "manf=%s";
-static const char BCMATTACHDATA(vstr_productname)[] = "productname=%s";
-static const char BCMATTACHDATA(vstr_manfid)[] = "manfid=0x%x";
-static const char BCMATTACHDATA(vstr_prodid)[] = "prodid=0x%x";
+static const char vstr_manf[] = "manf=%s";
+static const char vstr_productname[] = "productname=%s";
+static const char vstr_manfid[] = "manfid=0x%x";
+static const char vstr_prodid[] = "prodid=0x%x";
 #ifdef BCMSDIO
-static const char BCMATTACHDATA(vstr_sdmaxspeed)[] = "sdmaxspeed=%d";
-static const char BCMATTACHDATA(vstr_sdmaxblk)[][13] = {
+static const char vstr_sdmaxspeed[] = "sdmaxspeed=%d";
+static const char vstr_sdmaxblk[][13] = {
 "sdmaxblk0=%d", "sdmaxblk1=%d", "sdmaxblk2=%d"};
 #endif
-static const char BCMATTACHDATA(vstr_regwindowsz)[] = "regwindowsz=%d";
-static const char BCMATTACHDATA(vstr_sromrev)[] = "sromrev=%d";
-static const char BCMATTACHDATA(vstr_chiprev)[] = "chiprev=%d";
-static const char BCMATTACHDATA(vstr_subvendid)[] = "subvendid=0x%x";
-static const char BCMATTACHDATA(vstr_subdevid)[] = "subdevid=0x%x";
-static const char BCMATTACHDATA(vstr_boardrev)[] = "boardrev=0x%x";
-static const char BCMATTACHDATA(vstr_aa2g)[] = "aa2g=0x%x";
-static const char BCMATTACHDATA(vstr_aa5g)[] = "aa5g=0x%x";
-static const char BCMATTACHDATA(vstr_ag)[] = "ag%d=0x%x";
-static const char BCMATTACHDATA(vstr_cc)[] = "cc=%d";
-static const char BCMATTACHDATA(vstr_opo)[] = "opo=%d";
-static const char BCMATTACHDATA(vstr_pa0b)[][9] = {
+static const char vstr_regwindowsz[] = "regwindowsz=%d";
+static const char vstr_sromrev[] = "sromrev=%d";
+static const char vstr_chiprev[] = "chiprev=%d";
+static const char vstr_subvendid[] = "subvendid=0x%x";
+static const char vstr_subdevid[] = "subdevid=0x%x";
+static const char vstr_boardrev[] = "boardrev=0x%x";
+static const char vstr_aa2g[] = "aa2g=0x%x";
+static const char vstr_aa5g[] = "aa5g=0x%x";
+static const char vstr_ag[] = "ag%d=0x%x";
+static const char vstr_cc[] = "cc=%d";
+static const char vstr_opo[] = "opo=%d";
+static const char vstr_pa0b[][9] = {
 "pa0b0=%d", "pa0b1=%d", "pa0b2=%d"};
 
-static const char BCMATTACHDATA(vstr_pa0itssit)[] = "pa0itssit=%d";
-static const char BCMATTACHDATA(vstr_pa0maxpwr)[] = "pa0maxpwr=%d";
-static const char BCMATTACHDATA(vstr_pa1b)[][9] = {
+static const char vstr_pa0itssit[] = "pa0itssit=%d";
+static const char vstr_pa0maxpwr[] = "pa0maxpwr=%d";
+static const char vstr_pa1b[][9] = {
 "pa1b0=%d", "pa1b1=%d", "pa1b2=%d"};
 
-static const char BCMATTACHDATA(vstr_pa1lob)[][11] = {
+static const char vstr_pa1lob[][11] = {
 "pa1lob0=%d", "pa1lob1=%d", "pa1lob2=%d"};
 
-static const char BCMATTACHDATA(vstr_pa1hib)[][11] = {
+static const char vstr_pa1hib[][11] = {
 "pa1hib0=%d", "pa1hib1=%d", "pa1hib2=%d"};
 
-static const char BCMATTACHDATA(vstr_pa1itssit)[] = "pa1itssit=%d";
-static const char BCMATTACHDATA(vstr_pa1maxpwr)[] = "pa1maxpwr=%d";
-static const char BCMATTACHDATA(vstr_pa1lomaxpwr)[] = "pa1lomaxpwr=%d";
-static const char BCMATTACHDATA(vstr_pa1himaxpwr)[] = "pa1himaxpwr=%d";
-static const char BCMATTACHDATA(vstr_oem)[] =
+static const char vstr_pa1itssit[] = "pa1itssit=%d";
+static const char vstr_pa1maxpwr[] = "pa1maxpwr=%d";
+static const char vstr_pa1lomaxpwr[] = "pa1lomaxpwr=%d";
+static const char vstr_pa1himaxpwr[] = "pa1himaxpwr=%d";
+static const char vstr_oem[] =
     "oem=%02x%02x%02x%02x%02x%02x%02x%02x";
-static const char BCMATTACHDATA(vstr_boardflags)[] = "boardflags=0x%x";
-static const char BCMATTACHDATA(vstr_boardflags2)[] = "boardflags2=0x%x";
-static const char BCMATTACHDATA(vstr_ledbh)[] = "ledbh%d=0x%x";
-static const char BCMATTACHDATA(vstr_noccode)[] = "ccode=0x0";
-static const char BCMATTACHDATA(vstr_ccode)[] = "ccode=%c%c";
-static const char BCMATTACHDATA(vstr_cctl)[] = "cctl=0x%x";
-static const char BCMATTACHDATA(vstr_cckpo)[] = "cckpo=0x%x";
-static const char BCMATTACHDATA(vstr_ofdmpo)[] = "ofdmpo=0x%x";
-static const char BCMATTACHDATA(vstr_rdlid)[] = "rdlid=0x%x";
-static const char BCMATTACHDATA(vstr_rdlrndis)[] = "rdlrndis=%d";
-static const char BCMATTACHDATA(vstr_rdlrwu)[] = "rdlrwu=%d";
-static const char BCMATTACHDATA(vstr_usbfs)[] = "usbfs=%d";
-static const char BCMATTACHDATA(vstr_wpsgpio)[] = "wpsgpio=%d";
-static const char BCMATTACHDATA(vstr_wpsled)[] = "wpsled=%d";
-static const char BCMATTACHDATA(vstr_rdlsn)[] = "rdlsn=%d";
-static const char BCMATTACHDATA(vstr_rssismf2g)[] = "rssismf2g=%d";
-static const char BCMATTACHDATA(vstr_rssismc2g)[] = "rssismc2g=%d";
-static const char BCMATTACHDATA(vstr_rssisav2g)[] = "rssisav2g=%d";
-static const char BCMATTACHDATA(vstr_bxa2g)[] = "bxa2g=%d";
-static const char BCMATTACHDATA(vstr_rssismf5g)[] = "rssismf5g=%d";
-static const char BCMATTACHDATA(vstr_rssismc5g)[] = "rssismc5g=%d";
-static const char BCMATTACHDATA(vstr_rssisav5g)[] = "rssisav5g=%d";
-static const char BCMATTACHDATA(vstr_bxa5g)[] = "bxa5g=%d";
-static const char BCMATTACHDATA(vstr_tri2g)[] = "tri2g=%d";
-static const char BCMATTACHDATA(vstr_tri5gl)[] = "tri5gl=%d";
-static const char BCMATTACHDATA(vstr_tri5g)[] = "tri5g=%d";
-static const char BCMATTACHDATA(vstr_tri5gh)[] = "tri5gh=%d";
-static const char BCMATTACHDATA(vstr_rxpo2g)[] = "rxpo2g=%d";
-static const char BCMATTACHDATA(vstr_rxpo5g)[] = "rxpo5g=%d";
-static const char BCMATTACHDATA(vstr_boardtype)[] = "boardtype=0x%x";
-static const char BCMATTACHDATA(vstr_leddc)[] = "leddc=0x%04x";
-static const char BCMATTACHDATA(vstr_vendid)[] = "vendid=0x%x";
-static const char BCMATTACHDATA(vstr_devid)[] = "devid=0x%x";
-static const char BCMATTACHDATA(vstr_xtalfreq)[] = "xtalfreq=%d";
-static const char BCMATTACHDATA(vstr_txchain)[] = "txchain=0x%x";
-static const char BCMATTACHDATA(vstr_rxchain)[] = "rxchain=0x%x";
-static const char BCMATTACHDATA(vstr_antswitch)[] = "antswitch=0x%x";
-static const char BCMATTACHDATA(vstr_regrev)[] = "regrev=0x%x";
-static const char BCMATTACHDATA(vstr_antswctl2g)[] = "antswctl2g=0x%x";
-static const char BCMATTACHDATA(vstr_triso2g)[] = "triso2g=0x%x";
-static const char BCMATTACHDATA(vstr_pdetrange2g)[] = "pdetrange2g=0x%x";
-static const char BCMATTACHDATA(vstr_extpagain2g)[] = "extpagain2g=0x%x";
-static const char BCMATTACHDATA(vstr_tssipos2g)[] = "tssipos2g=0x%x";
-static const char BCMATTACHDATA(vstr_antswctl5g)[] = "antswctl5g=0x%x";
-static const char BCMATTACHDATA(vstr_triso5g)[] = "triso5g=0x%x";
-static const char BCMATTACHDATA(vstr_pdetrange5g)[] = "pdetrange5g=0x%x";
-static const char BCMATTACHDATA(vstr_extpagain5g)[] = "extpagain5g=0x%x";
-static const char BCMATTACHDATA(vstr_tssipos5g)[] = "tssipos5g=0x%x";
-static const char BCMATTACHDATA(vstr_maxp2ga0)[] = "maxp2ga0=0x%x";
-static const char BCMATTACHDATA(vstr_itt2ga0)[] = "itt2ga0=0x%x";
-static const char BCMATTACHDATA(vstr_pa)[] = "pa%dgw%da%d=0x%x";
-static const char BCMATTACHDATA(vstr_pahl)[] = "pa%dg%cw%da%d=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5ga0)[] = "maxp5ga0=0x%x";
-static const char BCMATTACHDATA(vstr_itt5ga0)[] = "itt5ga0=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5gha0)[] = "maxp5gha0=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5gla0)[] = "maxp5gla0=0x%x";
-static const char BCMATTACHDATA(vstr_maxp2ga1)[] = "maxp2ga1=0x%x";
-static const char BCMATTACHDATA(vstr_itt2ga1)[] = "itt2ga1=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5ga1)[] = "maxp5ga1=0x%x";
-static const char BCMATTACHDATA(vstr_itt5ga1)[] = "itt5ga1=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5gha1)[] = "maxp5gha1=0x%x";
-static const char BCMATTACHDATA(vstr_maxp5gla1)[] = "maxp5gla1=0x%x";
-static const char BCMATTACHDATA(vstr_cck2gpo)[] = "cck2gpo=0x%x";
-static const char BCMATTACHDATA(vstr_ofdm2gpo)[] = "ofdm2gpo=0x%x";
-static const char BCMATTACHDATA(vstr_ofdm5gpo)[] = "ofdm5gpo=0x%x";
-static const char BCMATTACHDATA(vstr_ofdm5glpo)[] = "ofdm5glpo=0x%x";
-static const char BCMATTACHDATA(vstr_ofdm5ghpo)[] = "ofdm5ghpo=0x%x";
-static const char BCMATTACHDATA(vstr_cddpo)[] = "cddpo=0x%x";
-static const char BCMATTACHDATA(vstr_stbcpo)[] = "stbcpo=0x%x";
-static const char BCMATTACHDATA(vstr_bw40po)[] = "bw40po=0x%x";
-static const char BCMATTACHDATA(vstr_bwduppo)[] = "bwduppo=0x%x";
-static const char BCMATTACHDATA(vstr_mcspo)[] = "mcs%dgpo%d=0x%x";
-static const char BCMATTACHDATA(vstr_mcspohl)[] = "mcs%dg%cpo%d=0x%x";
-static const char BCMATTACHDATA(vstr_custom)[] = "customvar%d=0x%x";
-static const char BCMATTACHDATA(vstr_cckdigfilttype)[] = "cckdigfilttype=%d";
-static const char BCMATTACHDATA(vstr_boardnum)[] = "boardnum=%d";
-static const char BCMATTACHDATA(vstr_macaddr)[] = "macaddr=%s";
-static const char BCMATTACHDATA(vstr_usbepnum)[] = "usbepnum=0x%x";
-static const char BCMATTACHDATA(vstr_end)[] = "END\0";
+static const char vstr_boardflags[] = "boardflags=0x%x";
+static const char vstr_boardflags2[] = "boardflags2=0x%x";
+static const char vstr_ledbh[] = "ledbh%d=0x%x";
+static const char vstr_noccode[] = "ccode=0x0";
+static const char vstr_ccode[] = "ccode=%c%c";
+static const char vstr_cctl[] = "cctl=0x%x";
+static const char vstr_cckpo[] = "cckpo=0x%x";
+static const char vstr_ofdmpo[] = "ofdmpo=0x%x";
+static const char vstr_rdlid[] = "rdlid=0x%x";
+static const char vstr_rdlrndis[] = "rdlrndis=%d";
+static const char vstr_rdlrwu[] = "rdlrwu=%d";
+static const char vstr_usbfs[] = "usbfs=%d";
+static const char vstr_wpsgpio[] = "wpsgpio=%d";
+static const char vstr_wpsled[] = "wpsled=%d";
+static const char vstr_rdlsn[] = "rdlsn=%d";
+static const char vstr_rssismf2g[] = "rssismf2g=%d";
+static const char vstr_rssismc2g[] = "rssismc2g=%d";
+static const char vstr_rssisav2g[] = "rssisav2g=%d";
+static const char vstr_bxa2g[] = "bxa2g=%d";
+static const char vstr_rssismf5g[] = "rssismf5g=%d";
+static const char vstr_rssismc5g[] = "rssismc5g=%d";
+static const char vstr_rssisav5g[] = "rssisav5g=%d";
+static const char vstr_bxa5g[] = "bxa5g=%d";
+static const char vstr_tri2g[] = "tri2g=%d";
+static const char vstr_tri5gl[] = "tri5gl=%d";
+static const char vstr_tri5g[] = "tri5g=%d";
+static const char vstr_tri5gh[] = "tri5gh=%d";
+static const char vstr_rxpo2g[] = "rxpo2g=%d";
+static const char vstr_rxpo5g[] = "rxpo5g=%d";
+static const char vstr_boardtype[] = "boardtype=0x%x";
+static const char vstr_leddc[] = "leddc=0x%04x";
+static const char vstr_vendid[] = "vendid=0x%x";
+static const char vstr_devid[] = "devid=0x%x";
+static const char vstr_xtalfreq[] = "xtalfreq=%d";
+static const char vstr_txchain[] = "txchain=0x%x";
+static const char vstr_rxchain[] = "rxchain=0x%x";
+static const char vstr_antswitch[] = "antswitch=0x%x";
+static const char vstr_regrev[] = "regrev=0x%x";
+static const char vstr_antswctl2g[] = "antswctl2g=0x%x";
+static const char vstr_triso2g[] = "triso2g=0x%x";
+static const char vstr_pdetrange2g[] = "pdetrange2g=0x%x";
+static const char vstr_extpagain2g[] = "extpagain2g=0x%x";
+static const char vstr_tssipos2g[] = "tssipos2g=0x%x";
+static const char vstr_antswctl5g[] = "antswctl5g=0x%x";
+static const char vstr_triso5g[] = "triso5g=0x%x";
+static const char vstr_pdetrange5g[] = "pdetrange5g=0x%x";
+static const char vstr_extpagain5g[] = "extpagain5g=0x%x";
+static const char vstr_tssipos5g[] = "tssipos5g=0x%x";
+static const char vstr_maxp2ga0[] = "maxp2ga0=0x%x";
+static const char vstr_itt2ga0[] = "itt2ga0=0x%x";
+static const char vstr_pa[] = "pa%dgw%da%d=0x%x";
+static const char vstr_pahl[] = "pa%dg%cw%da%d=0x%x";
+static const char vstr_maxp5ga0[] = "maxp5ga0=0x%x";
+static const char vstr_itt5ga0[] = "itt5ga0=0x%x";
+static const char vstr_maxp5gha0[] = "maxp5gha0=0x%x";
+static const char vstr_maxp5gla0[] = "maxp5gla0=0x%x";
+static const char vstr_maxp2ga1[] = "maxp2ga1=0x%x";
+static const char vstr_itt2ga1[] = "itt2ga1=0x%x";
+static const char vstr_maxp5ga1[] = "maxp5ga1=0x%x";
+static const char vstr_itt5ga1[] = "itt5ga1=0x%x";
+static const char vstr_maxp5gha1[] = "maxp5gha1=0x%x";
+static const char vstr_maxp5gla1[] = "maxp5gla1=0x%x";
+static const char vstr_cck2gpo[] = "cck2gpo=0x%x";
+static const char vstr_ofdm2gpo[] = "ofdm2gpo=0x%x";
+static const char vstr_ofdm5gpo[] = "ofdm5gpo=0x%x";
+static const char vstr_ofdm5glpo[] = "ofdm5glpo=0x%x";
+static const char vstr_ofdm5ghpo[] = "ofdm5ghpo=0x%x";
+static const char vstr_cddpo[] = "cddpo=0x%x";
+static const char vstr_stbcpo[] = "stbcpo=0x%x";
+static const char vstr_bw40po[] = "bw40po=0x%x";
+static const char vstr_bwduppo[] = "bwduppo=0x%x";
+static const char vstr_mcspo[] = "mcs%dgpo%d=0x%x";
+static const char vstr_mcspohl[] = "mcs%dg%cpo%d=0x%x";
+static const char vstr_custom[] = "customvar%d=0x%x";
+static const char vstr_cckdigfilttype[] = "cckdigfilttype=%d";
+static const char vstr_boardnum[] = "boardnum=%d";
+static const char vstr_macaddr[] = "macaddr=%s";
+static const char vstr_usbepnum[] = "usbepnum=0x%x";
+static const char vstr_end[] = "END\0";
 
 u8 patch_pair;
 
--- a/drivers/staging/brcm80211/util/hndpmu.c
+++ b/drivers/staging/brcm80211/util/hndpmu.c
@@ -308,7 +308,7 @@ typedef struct {
 #define RES_DEPEND_ADD		1	/* Add to the  dependancies mask */
 #define RES_DEPEND_REMOVE	-1	/* Remove from the dependancies mask */
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4328a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4328a0_res_updown[] = {
 	{
 	RES4328_EXT_SWITCHER_PWM, 0x0101}, {
 	RES4328_BB_SWITCHER_PWM, 0x1f01}, {
@@ -332,7 +332,7 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4328_BB_PLL_PU, 0x0701}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4328a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4328a0_res_depend[] = {
 	/* Adjust ILP request resource not to force ext/BB switchers into burst mode */
 	{
 	PMURES_BIT(RES4328_ILP_REQUEST),
@@ -341,7 +341,7 @@ static const pmu_res_depend_t BCMATTACHD
 		    PMURES_BIT(RES4328_BB_SWITCHER_PWM), NULL}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4325a0_res_updown_qt)[] = {
+static const pmu_res_updown_t bcm4325a0_res_updown_qt[] = {
 	{
 	RES4325_HT_AVAIL, 0x0300}, {
 	RES4325_BBPLL_PWRSW_PU, 0x0101}, {
@@ -353,12 +353,12 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4325_CBUCK_PWM, 0x0803}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4325a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4325a0_res_updown[] = {
 	{
 	RES4325_XTAL_PU, 0x1501}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4325a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4325a0_res_depend[] = {
 	/* Adjust OTP PU resource dependencies - remove BB BURST */
 	{
 	PMURES_BIT(RES4325_OTP_PU),
@@ -399,7 +399,7 @@ static const pmu_res_depend_t BCMATTACHD
 		    PMURES_BIT(RES4325B0_CBUCK_PWM), si_pmu_res_depfltr_ncb}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4315a0_res_updown_qt)[] = {
+static const pmu_res_updown_t bcm4315a0_res_updown_qt[] = {
 	{
 	RES4315_HT_AVAIL, 0x0101}, {
 	RES4315_XTAL_PU, 0x0100}, {
@@ -411,12 +411,12 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4315_CBUCK_LPOM, 0x0100}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4315a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4315a0_res_updown[] = {
 	{
 	RES4315_XTAL_PU, 0x2501}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4315a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4315a0_res_depend[] = {
 	/* Adjust OTP PU resource dependencies - not need PALDO unless write */
 	{
 	PMURES_BIT(RES4315_OTP_PU),
@@ -482,7 +482,7 @@ static const pmu_res_depend_t BCMINITDAT
 		    PMURES_BIT(RES4329_BBPLL_PWRSW_PU), NULL}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4319a0_res_updown_qt)[] = {
+static const pmu_res_updown_t bcm4319a0_res_updown_qt[] = {
 	{
 	RES4319_HT_AVAIL, 0x0101}, {
 	RES4319_XTAL_PU, 0x0100}, {
@@ -494,12 +494,12 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4319_CBUCK_LPOM, 0x0100}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4319a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4319a0_res_updown[] = {
 	{
 	RES4319_XTAL_PU, 0x3f01}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4319a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4319a0_res_depend[] = {
 	/* Adjust OTP PU resource dependencies - not need PALDO unless write */
 	{
 	PMURES_BIT(RES4319_OTP_PU),
@@ -521,7 +521,7 @@ static const pmu_res_depend_t BCMATTACHD
 		    PMURES_BIT(RES4319_AFE_PWRSW_PU), NULL}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4336a0_res_updown_qt)[] = {
+static const pmu_res_updown_t bcm4336a0_res_updown_qt[] = {
 	{
 	RES4336_HT_AVAIL, 0x0101}, {
 	RES4336_XTAL_PU, 0x0100}, {
@@ -531,18 +531,18 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4336_CBUCK_LPOM, 0x0100}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4336a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4336a0_res_updown[] = {
 	{
 	RES4336_HT_AVAIL, 0x0D01}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4336a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4336a0_res_depend[] = {
 	/* Just a dummy entry for now */
 	{
 	PMURES_BIT(RES4336_RSVD), RES_DEPEND_ADD, 0, NULL}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4330a0_res_updown_qt)[] = {
+static const pmu_res_updown_t bcm4330a0_res_updown_qt[] = {
 	{
 	RES4330_HT_AVAIL, 0x0101}, {
 	RES4330_XTAL_PU, 0x0100}, {
@@ -552,12 +552,12 @@ static const pmu_res_updown_t BCMATTACHD
 	RES4330_CBUCK_LPOM, 0x0100}
 };
 
-static const pmu_res_updown_t BCMATTACHDATA(bcm4330a0_res_updown)[] = {
+static const pmu_res_updown_t bcm4330a0_res_updown[] = {
 	{
 	RES4330_HT_AVAIL, 0x0e02}
 };
 
-static const pmu_res_depend_t BCMATTACHDATA(bcm4330a0_res_depend)[] = {
+static const pmu_res_depend_t bcm4330a0_res_depend[] = {
 	/* Just a dummy entry for now */
 	{
 	PMURES_BIT(RES4330_HT_AVAIL), RES_DEPEND_ADD, 0, NULL}
